Godot_test/ClientScripts/TopMenu.cs

```diff
@@ -1,26 +1,59 @@
 using Godot;
+using Roguelike.Abstract;
 using System;
+using System.IO;
 
 public partial class TopMenu : Control
 {
-	private async void _on_exit_pressed()
+  public override void _Ready()
+  {
+		string path = System.IO.Path.GetTempPath() + "Roguelike";
```

Comment:
Add a class SavedGames to Roguelike\Serialization and have there method returning list of saved games.
Add ut in RoguelikeUnitTests.Core/SerializationTests.cs showing it works.